js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+      multiAudioCol.style.width = maxWidth + 'px';
+      closeCaptionsCol.style.width = maxWidth + 'px';
+    }
+  },
+  /**
+   *
+   * @param languages {Array} - list of available languages
+   * @param language {String} - a selected language
+   * @returns {Array<{id: String, label: String, enabled: Boolean}>} an array of languages info objects
+   * @private
+   */
+  getClosedCaptions: function(languages, language) {
+    var closeCaptions = [];
+    if (Array.isArray(languages)) {
+      for (var index = 0; index < languages.length; index++) {
+        var isSelectedCc = languages[index] === language;
+        var cc = {
+          id: languages[index],
+          label: languages[index],
+          enabled: isSelectedCc
+        };
+        closeCaptions.push(cc);
+      }
+    }
+    return closeCaptions;
+  },
+
+  /**
+   * when clicking on an item from an cc list, set the corresponding cc value
+   * @param id {string} - id of clicked element
+   */
+  handleClickCC: function(id) {
+    if (this.props.controller && typeof this.props.controller.onClosedCaptionChange === 'function') {
+      this.props.controller.onClosedCaptionChange('language', id);
+    }
+  },
+
+  /**
+   * when clicking on an item from an audio list, set the corresponding audio value
+   * @param id {string} - id of clicked element
+   */
+  handleClickMA: function(id) {
+    if (this.props.controller && typeof this.props.controller.setCurrentAudio === 'function') {
+      this.props.controller.setCurrentAudio(id);
+    }
+  },
+
+  render: function() {
+    var multiAudioCol = null;
+    var closeCaptionsCol = null;
+    if (
+      this.props.controller &&
+      this.props.controller.state &&
+      this.props.controller.state.multiAudio &&
+      this.props.controller.state.multiAudio.tracks &&
+      this.props.controller.state.multiAudio.tracks.length > 0
+    ) {
+      multiAudioCol = (
+        <MultiAudioTab
+          ref="multiAudioCol"
+          handleClick={this.handleClickMA}
+          skinConfig={this.props.skinConfig}
+          list={this.props.controller.state.multiAudio.tracks}
+        />
+      );
+    }
+    if (
+      this.props.controller &&
+      this.props.controller.state &&
+      this.props.controller.state.closedCaptionOptions &&
+      this.props.controller.state.closedCaptionOptions.availableLanguages &&
+      this.props.controller.state.closedCaptionOptions.availableLanguages.languages &&
+      this.props.controller.state.closedCaptionOptions.availableLanguages.languages.length > 0
+    ) {
+      var closedCaptions = this.getClosedCaptions(
+        this.props.controller.state.closedCaptionOptions.availableLanguages.languages,
+        this.props.controller.state.closedCaptionOptions.language
+      );
+      closeCaptionsCol = (
+        <Tab
+          ref="closeCaptionsCol"
+          handleClick={this.handleClickCC}
+          skinConfig={this.props.skinConfig}
+          list={closedCaptions}
```

Comment:
Can we use something more descriptive than "list"? How about captions?

Reply:
I think `captions` won't work here since it's a generic `Tab` component that wraps both multiAudio and closedCaptions. We could go with `itemsList` thought, what do you think?

Reply:
Changed to `itemsList`.